otherlibs/win32unix/open.c

```diff
@@ -73,9 +77,11 @@ CAMLprim value unix_open(value path, value flags, value perm)
                       : cloexec & KEEPEXEC ? TRUE
                                            : !unix_cloexec_default;
 
-  h = CreateFile(String_val(path), fileaccess,
-                 sharemode, &attr,
-                 filecreate, fileattrib, NULL);
+  wpath = caml_stat_strdup_to_utf16(String_val(path));
+  h = CreateFileW(wpath, fileaccess,
+                  sharemode, &attr,
+                  filecreate, fileattrib, NULL);
```

Comment:
Here I thought that explicitly using the "W" functions was more robust, as the code would continue to compile correctly even if `UNICODE` is not defined.  But I see that the Microsoft docs also recommend using the unsuffixed version, so will do that.